### ISSUE TITLE
Support for batch effects, confounding factors, etc.?

### ISSUE THREAD

**Issue by DarwinAwardWinner** [dcjones/isolator#3]
Isolator looks like an interesting new approach, but most of my experiments cannot be described simply as disjoint groups of samples. Almost always, there are one or more other factors that I want to include in the model as well. Are there plans to support more complex designs, ideally arbitrary design matrices, in the future?

**Comment by dcjones** [comment]
Yes! The two biggest shortcomings with isolator are the rigidity of the model and difficultly scaling to large numbers of samples. I'm working on a new isoform quantification tool and experimenting with approaches to address both these issues, but it'll probably be a while until I make that public.

**Comment by rob-p** [comment]
Hi guys,

  I'm justing wondering if the new tool is [polee](https://github.com/dcjones/polee), which @dcjones just made public.  It looks _very cool_, and I'm excited to learn more about it!

**Comment by dcjones** [comment]
Yeah, polee is the tool I was referencing here, thanks for noticing. It does more or less what I was trying to do with isolator, but can handle a very large number of samples and it does regression with arbitrary design matrices (and makes it much easier to implement other types of models).

I have a paper submitted that I'll try to post to biorxiv soon.
